admin-dev/themes/new-theme/template/controllers/orders/page_header_toolbar.tpl

```diff
@@ -37,3 +38,12 @@
     {$title}
   {/if}
 {/block}
+
+<div class="header-toolbar d-print-none">
```

Comment:
I don't understand this new block? It wasn't there before and it's not used by any layout so what's its purpose?

Reply:
It was a trick used to obtain this display:
<img src="https://user-images.githubusercontent.com/3830050/69636793-aedade80-1057-11ea-85bd-554da34f8c45.png">
However I reverted this trick (see https://github.com/PrestaShop/PrestaShop/pull/16560) as it was creating side-effects (see https://github.com/PrestaShop/PrestaShop/issues/16549)